tests/validation/tests/v3_api/test_airgap_ha.py

```diff
@@ -0,0 +1,502 @@
+        bastion_node, push_images=True, noauth_reg_name=""):
+    get_images_command = \
+        'wget -O rancher-images.txt https://github.com/rancher/rancher/' \
+        'releases/download/{0}/rancher-images.txt && ' \
+        'wget -O rancher-save-images.sh https://github.com/rancher/rancher/' \
+        'releases/download/{0}/rancher-save-images.sh && ' \
+        'wget -O rancher-load-images.sh https://github.com/rancher/rancher/' \
+        'releases/download/{0}/rancher-load-images.sh'.format(
+            RANCHER_SERVER_VERSION)
+    bastion_node.execute_command(get_images_command)
+
+    # Remove the "docker save" and "docker load" lines to save time
+    edit_save_and_load_command = \
+        "sudo sed -i '58d' rancher-save-images.sh && " \
+        "sudo sed -i '76d' rancher-load-images.sh && " \
+        "chmod +x rancher-save-images.sh && chmod +x rancher-load-images.sh"
+    bastion_node.execute_command(edit_save_and_load_command)
+
+    save_images_command = \
+        "./rancher-save-images.sh --image-list ./rancher-images.txt"
+    save_res = bastion_node.execute_command(save_images_command)
+
+    if push_images:
+        if len(noauth_reg_name) == 0:
+            load_images_command = \
+                "docker login {} -u {} -p {} && " \
+                "./rancher-load-images.sh --image-list ./rancher-images.txt " \
+                "--registry {}".format(
+                    bastion_node.host_name, PRIVATE_REGISTRY_USERNAME,
+                    PRIVATE_REGISTRY_PASSWORD, bastion_node.host_name)
+            load_res = bastion_node.execute_command(load_images_command)
+        else:
+            load_images_command = \
+                "./rancher-load-images.sh --image-list ./rancher-images.txt " \
+                "--registry {}".format(noauth_reg_name)
+            load_res = bastion_node.execute_command(load_images_command)
+        print(load_res)
+    else:
+        load_res = None
+
+    return save_res, load_res
+
+
+def prepare_airgap_node(bastion_node, number_of_nodes):
+    node_name = AG_HOST_NAME + "-agha"
+    # Create Airgap Node in AWS
+    ag_nodes = AmazonWebServices().create_multiple_nodes(
+        number_of_nodes, node_name, ami=AWS_AMI, public_ip=False)
+
+    for num, ag_node in enumerate(ag_nodes):
+        # Update docker for the user in node
+        ag_node_update_docker = \
+            'ssh -i "{}.pem" -o StrictHostKeyChecking=no {}@{} ' \
+            '"sudo usermod -aG docker {}"'.format(
+                bastion_node.ssh_key_name, AWS_USER,
+                ag_node.private_ip_address, AWS_USER)
+        bastion_node.execute_command(ag_node_update_docker)
+
+    return ag_nodes
+
+
+def setup_airgap_rancher(bastion_node, number_of_nodes=NUMBER_OF_INSTANCES):
+    # remove old setup files, if any
+    bastion_node.execute_command("rm ingress* config.* kube_config_* tls*")
+    # need an odd number of nodes for HA setup
+    if number_of_nodes % 2 == 0:
+        number_of_nodes += 1
```

Comment:
If we are manipulating the parameter set for number of instances we should print it so the user knows that was ammended

Reply:
added a print statement for these